proposed/http-client/http-client.md

```diff
@@ -73,17 +73,14 @@ use Psr\Http\Message\RequestInterface;
 use Psr\Http\Client\Exception;
 
 /**
- * Exception for when a request failed, providing access to the failed request.
- *
- * This could be due to an invalid request, example when you can seek in the request body stream
```

Comment:
I don't intend to remove this. But I simply cannot understand this sentence. :stuck_out_tongue_closed_eyes: